website/docs/architecture/chaos-execution-plane.md

```diff
@@ -0,0 +1,42 @@
+---
+id: chaos-execution-plane
+title: Chaos Execution Plane
+sidebar_label: Chaos Execution Plane
+---
+
+---
+
+<img src={require("../assets/execution-plane.png").default} alt="Chaos Execution Plane" />
+
+Chaos Execution Plane contains the components responsible for orchestrating the chaos injection in the target resources. They get installed in either an external target cluster if an external agent is being used or in the host cluster containing the control plane if a self-agent is being used. It can be further segregated into Litmus Agent Infrastructure components and Litmus Backend Execution Infrastructure components. 
```

Comment:
What is the preferred term here @ajeshbaby - external agent or remote agent ?

Reply:
It has been mentioned as "external agent" in the `Getting Started > Resources` section of the docs as well, hence I used the same term here.